templates/launch.html

```diff
@@ -160,21 +172,71 @@ <h3>Runner Data</h3>
         document.querySelector('#survey_metadata_fields').innerHTML = ""
     }
 
+    function showAllDivs() {
+        document.querySelector("#launch_pattern").setAttribute("disabled", "disabled")
+
+        document.querySelectorAll("#survey_metadata, .hidden-field-container, .group-title", ).forEach(function (el) {
+            el.style.display = 'block';
+        });
```

Comment:
I would use same convention throughout the file so for every `forEach` use `el` or `element` but not both at the same time:
```suggestion
        document.querySelectorAll("#survey_metadata, .hidden-field-container, .group-title", ).forEach(function (element) {
            element.style.display = 'block';
        });
```